schemas/search.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: search
+  title: Search
+  group: 2
+  short: Fields to describe details of a search event.
+  description: >
+    The Search fields describe information about a search request event: query or pagination. These fields should be used with the corresponding `event.action` set with the corresponding event action: `['search.query', 'search.page']`. Other fields that should be used with this field set include: `event.duration` to describe the duration of a search request, `event.created` to record the event's timestamp and optionally `source` fields to record context information such as `user.id` or `geo`.
```

Comment:
In ECS, the content of `event.action` is never mandated. It's meant to be defined by the source of the event. So the guidance you're providing here belongs in your project, not in ECS.

What we need to think about for `event.action`, is that other projects unrelated to yours may want to leverage the `search` field set, but use slightly different event actions that are more tailored to their use case.

Reply:
Love the guidance of calling out other general ECS fields that should be used in conjunction with the `search` field set 👍

Reply:
I'm going to change that wording a bit then to be more suggestive rather than prescriptive.